dds_registration/settings.py

```
@@ -45,17 +45,18 @@
     DEBUG=(bool, True),  # LOCAL  # and DEV
     SECRET_KEY=(str, 'django-insecure-1d*alw^8-nya9h#xhfjqe*5%w8!o7vy8!211ez++h!p_*nm%21'),
     SENDGRID_API_KEY=(str, ''),
+    REGISTRATION_SALT=(str, 'OdUqhiXaWN'),  # Random salt for registration keys (to use in django_registration module)
```

Please remove the, we don't ever want to commit things like this as we could forget to supply them locally at one point. I want to raise an error in that case.

---

It's already fixed in PR #43 -- changes were already integrated there and overrode with later changes. Just for case rebased actual main branch back into this one.